Exploration Queries/SecurityEvent/ParentProcessesOnHost.txt

```diff
@@ -1,20 +1,16 @@
-// Name: Parent Processes running on Host
-// Description: Any 4688 event that contains Parent Process running on a given host during the time of or recently after compromise and in the range of +-10m
-// Entity: #Host
-// Input: Computername, Hostname, HostId
-// Output: Processes
-//
-// QueryPeriod: -1h and +30m default, change as needed, not all event 4688 have Parent Process available
-//
-// Data Source: #SecurityEvent
-//
+// Id: 992b9912-a11e-487d-83b3-1d1ba6081746
+// DisplayName: Parent Processes running on Host
+// Description: Any 4688 event that contains Parent Process running on a given host during the time of or recently after compromise (set time range to +- 10m when running the query)
+// InputEntityType: Host
+// InputFields: [HostName]
+// OutputEntityTypes: [Process]
+// QueryPeriodBefore: 10m
+// QueryPeriodAfter: 10m
+// DataSources: #SecurityEvent
```

Comment:
Changed in order to maintain the GitHub searching functionality.

In case more than 1 DataSource is being used, it would become:
// DataSources: #SecurityEvent, #DnsEvents
[i.e. comma separated]